### ISSUE TITLE
password option

### ISSUE THREAD

**Issue by ghost** [triniwiz/nativescript-zip#32]
is it possible to create or open zip files which are protected with a password?

**Comment by triniwiz** [comment]
Yes use `password` https://github.com/triniwiz/nativescript-zip/blob/master/src/common.ts#L6
